pynestml/codegeneration/resources_nest/point_neuron/common/NeuronClass.jinja2

```diff
@@ -512,6 +512,7 @@ double {{neuronName}}::get_delayed_{{variable.get_symbol_name()}}() const
 
 void {{neuronName}}::update(nest::Time const & origin,const long from, const long to)
 {
+  auto get_t = [](){ return nest::Time( nest::Time::step( nest::kernel().simulation_manager.get_slice_origin().get_steps() ) ).get_ms(); };
```

Comment:
Perhaps this should be a function of the ``origin`` parameter (+ lag) rather than making a call to the kernel.